mysql-test/suite/sys_vars/r/path_priv.result

```diff
@@ -0,0 +1,18 @@
+#
+# MDEV-34391: Check privilige for setting global path
+#
+CREATE USER priv_test;
+connect  conn1,localhost,priv_test,,;
+connection conn1;
+SET @@global.path= 'sys';
+ERROR 42000: Access denied; you need (at least one of) the SUPER privilege(s) for this operation
```

Comment:
I wouldn't even bother testing that. You didn't write any code related to privileges, it's common behavior for all global variables, not specific to @@path. No need to test it for every new variable, it's always the same.